testsuite/tests/conftest.py

```
@@ -64,9 +64,64 @@ def pytest_runtest_setup(item):
+            exc_message = "; ".join(errs)
+
+    tb_lines.append(f"E   {call.excinfo.type.__name__}: {exc_message}")
+    return "\n".join(tb_lines)
+
+
+def _collect_rerun_attempt(item, call, report):
+    """Record failure message and captured output for one rerun attempt."""
+    message = _format_failure_message(call, report)
+    if report.failed:
+        report.longrepr = message
+    item.rerun_messages = getattr(item, "rerun_messages", []) + [message]
```

Ruff (0.15.10)

[warning] Consider `[*getattr(item, "rerun_messages", []), message]` instead of concatenation

Replace with `[*getattr(item, "rerun_messages", []), message]`

(RUF005)